packages/login/package.json

```diff
@@ -126,5 +126,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "resolutions": {
+    "react-router": "5.1.2",
+    "react-router-dom": "5.1.2",
+    "@types/react-router": "5.1.2",
+    "@types/react-router-dom": "5.1.2"
```

Comment:
Good questions. We tested few different alternatives, and there are couple of issues:

0. If you just run install it works.
1. Seems that our yarn version has known issue when resolving package versions in monorepo, and when adding another package resolving goes wrong.
https://github.com/yarnpkg/yarn/issues/8606 https://github.com/yarnpkg/yarn/issues/8068
2. Our setup is quite complex, adding `nohoisting` definition fixes the issue, with the cost of login tests receiving wrong react version. Adding `resolutions` fixes login tests but leaves the inital resolve issue open.

Since it is getting quite deep, we think the best worst strategy is
1. Use `nohoisting` instead of `resolutions`
2. Ignore login tests
3. Merge PR in
4. Merge develop to scopes
5. Immediately start updating login router. Since the root issue is caused by yarn's inability to resolve different versions of the same package consistently.
6. Take tests back